### PR TITLE
Add front-matter options to hide post meta and actions.

### PR DESCRIPTION
**Pull request by darekkay**
This makes custom 404 error pages look more clean (#327).

### Configuration

 - **Operating system with version** : 
 - **Node version** : 6.9.1
 - **Hexo version** : 3.2.0
 - **Hexo-cli version** :  1.0.2
 
### Changes proposed

 - Front-matter options to hide post meta and actions.

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -612,6 +612,8 @@ gallery:
     - http://i.imgur.com/o9r19kD.jpg "Dubai"
     - https://example.com/orignal.jpg https://example.com/thumbnail.jpg "Sidney"
 comments: false
+hideMeta: true
```
Comment (LouisBarranqueiro) [inline docs/user.md]: use `meta` and `actions` variable names to have the same code style as `comments`. so update your if condition with `(post.actions === undefined || post.actions)` to still show the meta and actions if variables are not declared in the front-matter.

File: docs/user.md [hunk 2]
```diff
@@ -798,7 +802,26 @@ E.g : `{% wide_image http://google.fr/images/image125.png "A beautiful sunrise"
...
+hideActions: true
+comments: false
+```
+
+Now you can customize your 404 error page just like any other blog post.
+Finally, you need to tell your server to use `/404.html` (which Hexo generates out of `404.md`) as your default 404 error page. Here are tutorials for the most common web servers:
+
+ - [Apache](https://www.digitalocean.com/community/tutorials/how-to-create-a-custom-404-page-in-apache)
+ - [Nginx](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-to-use-custom-error-pages-on-ubuntu-14-04)
```
Comment (LouisBarranqueiro) [inline docs/user.md]: Also add the link for Github pages tutorial: https://help.github.com/articles/creating-a-custom-404-page-for-your-github-pages-site/

Comment (jrbenito) [inline docs/user.md]: May worth to reference Amazon Cloudfront/S3 custom errors too: http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/custom-error-pages.html

**Comment by darekkay** [comment]
I've updated this PR :bowtie:

**Pull request merged**